plugins/techdocs/report.api.md

```diff
@@ -431,6 +431,7 @@ export const TechDocsReaderPageContent: (
 // @public
 export type TechDocsReaderPageContentProps = {
   entityRef?: CompoundEntityRef;
+  defaultPath?: string;
```

Comment:
We don't really need this do we?

The content [already has access](https://github.com/backstage/backstage/blob/674def926408995e0b589badb74f848a3ddef174/plugins/techdocs/src/reader/components/TechDocsReaderPageContent/TechDocsReaderPageContent.tsx#L91) to the entire entity being rendered (the `entityMetadata` value). Couldn't you already extract all that you need from inside this component without adding a new prop?